reference/5.1/Microsoft.PowerShell.Utility/Write-Information.md

```diff
@@ -166,6 +182,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## RELATED LINKS
 
+[about_Output_Streams](/powershell/module/Microsoft.PowerShell.Core/About/about_Output_Streams.md)
+
+[about_Redirection](/powershell/module/Microsoft.PowerShell.Core/About/about_Redirection.md)
```

Comment:
**Site Relative** links shouldn't include the extension.

Reply:
fixed